### DISCUSSION TITLE
Is the baseurl missing from assets/js/dist/sw.min.js?

### DISCUSSION THREAD

**Discussion by tnemoz** [cotes2020/jekyll-theme-chirpy#1909] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I'm trying to upgrade from 6.5.5 to 7.0.1. Everyting went more or less smoothly, until, when running `bundle exec jekyll serve`, I got a `ERROR /assets/js/data/swconf.js' not found`.

Looking at `assets/js/dist/sw.min.js` in my local `jekyll-theme-chirpy` gem, one can read the following:
```javascript
!function(){"use strict";importScripts("".concat("","/assets/js/data/swconf.js"));
```
I replaced it with
```javascript
!function(){"use strict";importScripts("".concat("","/tnemoz/assets/js/data/swconf.js"));
```
with `tnemoz` being my base url, and the error disappeared.

Did I do something wrong when setting up Jekyll, or is there an error in this file? For reference, the relevant part of my `_config.yml` is the following:
```yaml
# The base URL of your site
baseurl: "/tnemoz"
```
I'm quite new to this, so if this post misses crucial information, please tell me! And thanks in advance for your help :slightly_smiling_face: 

### Operations you have already tried

I tried modifying the file, which removed the error, but I think that I shouldn't normally manually modify these files, hence my concern.

**Comment by cotes2020** [comment, marked as answer]
Sorry, just found this discussion now. This is a bug, thanks for pointing it out.

**Reply by cotes2020** [reply]
Fixed in #1955
